packages/core/src/sso/OidcConnector/utils.ts

```diff
@@ -109,11 +109,12 @@ const issuedAtTimeTolerance = 600; // 10 minutes
 export const getIdTokenClaims = async (
   idToken: string,
   config: BaseOidcConfig,
-  nonceFromSession?: string
+  nonceFromSession: string | undefined,
+  skipIssuerCheck = false
 ) => {
   try {
     const { payload } = await jwtVerify(idToken, createRemoteJWKSet(new URL(config.jwksUri)), {
-      issuer: config.issuer,
+      issuer: skipIssuerCheck ? undefined : config.issuer,
```

Comment:
Instead of skipping, should we dynamically check the issuer instead? The current approach does not look secure.